pkg/rancher-desktop/backend/kube/lima.ts

```diff
@@ -126,9 +126,18 @@ export default class LimaKubernetesBackend extends events.EventEmitter implement
    */
   async install(config: BackendSettings, desiredVersion: semver.SemVer, allowSudo: boolean) {
     await this.progressTracker.action('Installing k3s', 50, async() => {
+      // installK3s removes old config and makes sure the directories are recreated
       await this.installK3s(desiredVersion);
-      await this.writeServiceScript(config, desiredVersion, allowSudo);
-      await BackendHelper.configureRuntimeClasses(this.vm);
+
+      const promises : Promise<void>[] = [this.writeServiceScript(config, desiredVersion, allowSudo)];
```

Comment:
```suggestion
      const promises: Promise<void>[] = [this.writeServiceScript(config, desiredVersion, allowSudo)];
```
This may also look nicer as:
```suggestion
      const promises: Promise<void>[] = [];

      promises.push(this.writeServiceScript(config, desiredVersion, allowSudo));
```